src/test/java/com/company/sqlTest/RecruitmentTest.java

```diff
@@ -0,0 +1,48 @@
+package com.company.sqlTest;
+
+import com.company.sqlTest.dataProvider.VacancySqlDataProvider;
+import com.company.sqlTest.pages.BaseTest;
+import com.company.sqlTest.pages.LoginPage;
+import com.company.sqlTest.pages.VacancyPage;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RecruitmentTest extends BaseTest {
+    LoginPage loginPage;
+
+
+    @Test
+    public void testRecruitment() {
+        ArrayList<String> map = new ArrayList<>();
+
+        VacancySqlDataProvider vacancySqlDataProvider = new VacancySqlDataProvider();
+        String vacancyFromSql = vacancySqlDataProvider.getVacanciesByEmployeeName("AQA Man", "Bilynska").toString();
```

Comment:
AQA Man - seems like vacancy name, but it has to be the output value  
i.e.  String vacancyFromSql = vacancySqlDataProvider.getVacanciesByEmployeeName("Olya", "Bilynska");
vacancyFromSql -> "AQA Man"